bitnami/jupyterhub/values.yaml

```diff
@@ -0,0 +1,1592 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+## @section Global parameters
+## @param global.imageRegistry Global Docker image registry
+## @param global.imagePullSecrets Global Docker registry secret names as an array
+##
+global:
+  imageRegistry:
+  imagePullSecrets: []
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
```

Comment:
In WordPress, I created a section for "Common Parameters", see https://github.com/bitnami/charts/blob/master/bitnami/wordpress/values.yaml#L22

Could we do the same here? Global != Common

Reply:
Done